### PR TITLE
Finished Vet Clinic part 3

### PR DESCRIPTION
**Pull request by Daf98**
# Vet Clinic III
- Added 'owners' table
- Added 'species' table
- Added foreign keys to relate newly created tables to original 'animals' table
- Performed queries using 'JOIN'
### What animals belong to Melody Pond?
image
### List of all animals that are pokemon (their type is Pokemon).
image
### List all owners and their animals, remember to include those that don't own any animal.
image
### How many animals are there per species?
image
### List all Digimon owned by Jennifer Orwell.
image
### List all animals owned by Dean Winchester that haven't tried to escape.
image
### Who owns the most animals?
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -10,3 +10,31 @@ weight_kg DEC NOT NULL
 );
 
 ALTER TABLE animals ADD species TEXT;
+
+CREATE TABLE owners(
+id INT GENERATED ALWAYS AS IDENTITY,
+full_name TEXT NOT NULL,
+age INT,
+PRIMARY KEY(id)
+);
```
Comment (mukhammadrizooff) [inline schema.sql]: - [x] [REQUIRED] Good job so far! On line 16 should like this `full_name varchar(255),` Please fix this.

File: schema.sql [hunk 2]
```diff
@@ -10,3 +10,31 @@ weight_kg DEC NOT NULL
...
+
+CREATE TABLE species(
+id INT GENERATED ALWAYS AS IDENTITY,
+name TEXT NOT NULL,
+PRIMARY KEY(id)
+);
```
Comment (mukhammadrizooff) [inline schema.sql]: - [x] [REQUIRED] Good job so far! On line 23 should like this `name varchar(255),` Please fix this.

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Daf98, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good implementations ✔️
- Gave correct values ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**